CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing Guidelines - Rocket.Chat.Java.SDK
+
+Great to have you here! Here are a few ways you can help make this project better!
+
+## Reporting an Issue
+
+[Github Issues](https://github.com/RocketChat/Rocket.Chat.Android/issues) are used to track todos, bugs, feature requests, and more.
```

Comment:
You are pointing to the Android repository

Reply:
Fixed